app/views/users/index.html.erb

```diff
@@ -1 +1,14 @@
-<h1>This is a list of users</h1>
\ No newline at end of file
+<section class="home-container">
+  <h1>Welcome to RoR - Blogs</h1>
+  <div class="users-container">
+    <% @users.each do |user| %>
+      <card class="user-item">
+        <img src='<%= user.photo %>'></img>
+        <div class="user-data">
+          <h2><%= user.name %></h2><br>
+          <h4 class="custom-card-t">Number of posts: <%= user.posts_counter %> </h4>
+        </div>
+      </card>
+    <% end %>
+  </div>
+</section>
```

Comment:
- [ ] It would be nice if you make your app more functional by adding links here to your user show page because i cannot go to your user_show page without changing the URL 😨 Kindly make each user a link so whenever a user is clicked then it takes us to the user_show page with respective user id 👍